WordPress/Classes/ViewRelated/Reader/ReaderStreamViewController.swift

```diff
@@ -1670,7 +1673,7 @@ extension ReaderStreamViewController : WPTableViewHandlerDelegate {
         }
 
         if recentlyBlockedSitePostObjectIDs.containsObject(post.objectID) {
```

Comment:
Technically it should be the same objectID, but for paranoia sake can this be `aPost.objectID` just for an extra dose of caution?